docs/src/pages/guides/composition/composition.md

```diff
@@ -53,42 +53,48 @@ The following example will render the `List` component with a `<nav>` element as
 </List>
 ```
 
-This pattern is very powerful and allows for great flexibility, as well as a way to interoperate with other libraries, such as [`react-router`](#react-router-demo) or your favorite forms library. But it also **comes with a small caveat!**
+This pattern is very powerful and allows for great flexibility, as well as a way to interoperate with other libraries, such as your favorite routing or forms library.
+But it also **comes with a small caveat!**
 
 ### Caveat with inlining
 
-Using an inline function as an argument for the `component` property may result in **unexpected unmounting**, since you pass a new component to the `component` property every time React renders.
+Using an inline function as an argument for the `component` prop may result in **unexpected unmounting**, since you pass a new component to the `component` prop every time React renders.
```

Comment:
```suggestion
Using an inline function as an argument for the `component` prop may result in **unexpected unmounting**, since a new component is passed every time React renders.
```